AUTHORS

```diff
@@ -1,4 +1,4 @@
-# This is the list of Virtual People ID Systems' significant contributors.
+# This is the list of Cross-Media Measurement's significant contributors.
```

Comment:
isn't this virtual people ID repo?

Reply:
Virtual People ID Systems is part of Cross Media Measurement project. We want to keep this aligned with other Cross Media Measurement repos.